tests/review_model_test.py

```diff
@@ -0,0 +1,33 @@
+import unittest
+from seed import app_with_seeded_data
+
```

Comment:
I was having trouble running tests with vscode until I did this instead. Can you check if this works for you? I think cuz it's running the tests from the root directory which only recognizes `tests.seed` rather than `seed`
```suggestion
from tests.seed import app_with_seeded_data
```